.codacy.yaml

```diff
@@ -0,0 +1,3 @@
+---
+exclude_paths:
+  - 'UnityProject/Assets/Plugins/YamlDotNet/**'
```

Comment:
I changed Codacy configuration to exclude the plugin folder altogether. You can get rid of this yaml.